Notes for Logseq.md

```diff
@@ -0,0 +1,60 @@
+# About the export for logseq #
+
+## How to export for logseq ##

```

Comment:
It would be even better if the required settings would be preset and readonly in the UI, if logSeqMode is set to true. In this case that toggle would be the main trigger of everything, and any problems could come from mixing the settings could be avoided.

Reply:
I agree with you that for obsedian or logseq, there should be an overall preset for each, upon selection, all the options be set properly automatically. But that would require a restructure of the program or sth. I can't change that here